layouts/_default/article.html

```diff
@@ -23,15 +23,15 @@ <h3 class="text-xl font-extrabold mt-3 md:mt-4 pb-2 "><a class="block h-auto lg:
         <div class="flex justify-between items-center px-6 py-3 border-t mt-6 xl:mt-8">
             <div class="flex items-center">
                 {{ $profile := .Params.profile }}
-                {{ if eq (len .Params.authors) 1}}
+                {{ if eq .Params.authors 1 }}
```

Comment:
This change was made to resolve an error that was encountered after using `.Related` in `blog/single.html`.